docusaurus/docs/e2e-test-a-plugin/migrate-from-grafana-e2e.md

```diff
@@ -0,0 +1,172 @@
+---
+id: migrate-from-grafana-e2e
+title: Migrate from @grafana/e2e
+description: How to migrate from @grafana/e2e to @grafana/plugin-e2e
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - testing
+  - e2e
+  - end-to-end
+  - migrate
+sidebar_position: 90
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ScaffoldPluginE2InstallNPM from '@snippets/plugin-e2e-install.npm.md';
+import ScaffoldPluginE2EInstallPNPM from '@snippets/plugin-e2e-install.pnpm.md';
+import ScaffoldPluginE2EInstallYarn from '@snippets/plugin-e2e-install.yarn.md';
+
+The [`@grafana/e2e`](https://www.npmjs.com/package/@grafana/e2e) tool will be depcrecated when Grafana 11.0.0 is released, and the support for this package will be dropped. We recommend all plugin authors to migrate their end-to-end tests to use Playwright and `@grafana/plugin-e2e` instead of Cypress and `@grafana/e2e`.
+
+In this guide you'll learn:
+
+- how to manually setup `@grafana/plugin-e2e` in your plugin
+- how to migrate tests
+- how to run Playwright tests in CI
+- how to uninstall Cypress and `@grafana/e2e`
+
+## Manually installing @grafana/plugin-e2e
+
+If you have a plugin that was not scaffolded with setup for `@grafana/plugin-e2e` and `@playwright/test`, you can follow these steps to set it up manually.
```

Comment:
This sentence doesn't quite read right for me. I'm not sure its necessary in all honesty. If you want to keep something like it, maybe  could change to something like:

> Plugins scaffolded with `create-plugin` v4.5.0 and greater automatically include configuration for `@grafana/plugin-e2e` and `@playwright/test`. To add this configuration manually, follow these steps: